sass/themes/cr/2019/components/navigation/_expanded-footer.scss

```diff
@@ -27,40 +27,39 @@ div.footer--minimal.footer-nav-outer-wrapper {
 
 div.footer--expanded.footer-nav-outer-wrapper {
   line-height: initial;
-  background: #f2f2f2;
-  padding: 45px 18px 40px;
+  background: $colour-black; 
```

Comment:
Based on the design, this needs to be #FFF,  this variable is from another theme

Reply:
Haha whoops, yes.. wrong way